src/standard/standard.android.ts

```diff
@@ -2,6 +2,15 @@ import { android as androidApp } from "tns-core-modules/application";
 import { Page } from "tns-core-modules/ui/page";
 import { StripeAddress, StripeConfigCommon, StripePaymentListener, StripePaymentMethod, StripeShippingAddressField, StripeShippingMethod } from "./standard.common";
 
+declare let global: any;
```

Comment:
I know very little about Android development so I can't evaluate these changes.

Since I'm having trouble running the demo apps on Android locally on my Mac, I wonder if these changes are related to the problems? (See my comment on the PR.)

Reply:
Regarding the android updates, it's due to the use of androidx in NS6. As we want to support old versions of NS, there's a fallback on the "old" accessor.

Example:
https://github.com/EddyVerbruggen/nativescript-barcodescanner/blob/bf5f974f61d1cf15f90ee4b0ae9d301258a3cf32/src/barcodescanner.android.ts#L10

Reply:
I've fixed all the above points. Regarding the `export 'StripeShippingAddressField' was not found`, It's due to the new webpack loader(cf: https://github.com/NativeScript/nativescript-dev-webpack/issues/927)

I've persisted the webpack configurations file on the demo repos